fv3core/utils/validation.py

```diff
@@ -0,0 +1,39 @@
+from typing import Tuple
+
+import numpy as np
+
+import fv3core.utils.gt4py_utils
+
+
+class SelectiveValidation:
```

Comment:
this should probably live in the fv3core/testing or tests/ folder rather than utils, as it is just used for tests